src/components/dapp-staking-v2/my-staking/MyDapps.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div>
+    <template v-if="width >= screenSize.lg">
+      <div class="table-container">
+        <table id="my-table">
+          <thead>
+            <tr>
+              <th>dApps</th>
+              <th>Staked Amount</th>
+              <th>Total Earned</th>
+              <th>Manage</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr v-for="t in items" :key="t.id">
+              <td>{{ t.name }}</td>
+              <td>{{ t.stakedAmount.toLocaleString() }} ASTR</td>
```

Comment:
You can change like [this](https://github.com/AstarNetwork/astar-apps/blob/feat/dapp-staking-v2/src/components/assets/NativeAssetList.vue#L28)